cryptoki/README.md

```diff
@@ -7,4 +7,77 @@
 
 This is the high-level, Rust idiomatic wrapper crate for PKCS #11.
 
+The items in this crate only expose idiomatic and safe Rust types and
+functions to interface with the PKCS11 API. All the PKCS11 items might
+not be implemented but everything that is implemented is safe.
+
+## Example
```

Comment:
The example is very nice! Wouldn't it be nice to copy it to a new `cryptoki/examples/rsa.rs` file (like we do in `tss-esapi`) so that the example is checked for compilation every time `cargo test` is run? That way we are sure it at least compiles as the crate evolves.

Here we could link to the respective example instead :)

Reply:
> so that the example is checked for compilation every time cargo test is run? That way we are sure it at least compiles as the crate evolves.

I'm not sure if it's completely clear but this example in the readme is *also* checked every time `cargo test` runs (due to the `doc include str` thing in `lib.rs`).

(Adding more examples, or even the same for redundancy, is :ok_hand: with me)

Reply:
ahhhh I did not know that, this is very neat!

Reply:
Haha, yes, I like it because it's both illustrative *and* checked. I try to do that in every crate I contribute to. Just today [I fixed a README.md example](https://github.com/matusf/z-base-32/pull/60) that was slightly broken because it didn't use this style :)